### PR TITLE
add postage stamp tests

### PR DESCRIPTION
**Pull request by ralph-pichler**
tests for postage stamp and price oracle

### CODE REVIEW

File: test/PostageStamp.test.ts [hunk 1]
```diff
@@ -42,22 +46,160 @@ describe('PostageStamp', function () {
...
+        await this.postageStamp.topUp(this.batch.id, this.topupAmount);
+        expect(await this.token.balanceOf(stamper)).to.equal(0);
+        expect(await this.token.balanceOf(this.postageStamp.address)).to.equal(
+          this.batch.initialPayment + this.topupAmount
+        );
+      });
+
+      it('should not top up non-existing batches', async function () {
+        await expect(
+          this.postageStamp.topUp(computeBatchId(deployer, this.batch.nonce), this.topupAmount)
```
Comment (paxthemax) [inline test/PostageStamp.test.ts:143]: Suggestion: extract `computeBatchId(deployer, this.batch.nonce)` to a variable such as `nonExistingBatchId`. That way it's quite clear what is going on regardless of the test setup. Alternatively, we could have a comment just to clarify.

File: test/PriceOracle.test.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+describe('PriceOracle', function () {
+  describe('when deploying contract', function () {
+    beforeEach(async function () {
+      await deployments.fixture();
+    });
+
+    it('should deploy PriceOracle', async function () {
+      const priceOracle = await ethers.getContract('PriceOracle');
+      expect(priceOracle.address).to.be.a('string');
```
Comment (paxthemax) [inline test/PriceOracle.test.ts:25]: Consider: `expect(priceOracle.address).to.be.properAddress`.

**Review by paxthemax — APPROVED** [review]
Approved. I raised some very minor points.

File: test/PostageStamp.test.ts (hunk 1 above)
Comment (ralph-pichler) [reply test/PostageStamp.test.ts:143]: fixed

File: test/PriceOracle.test.ts (hunk 2 above)
Comment (ralph-pichler) [reply test/PriceOracle.test.ts]: fixed here, and also for the postageStamp contract

**Pull request merged**